api/app/signals/apps/services/domain/permissions/signal.py

```diff
@@ -0,0 +1,78 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2021 Gemeente Amsterdam
+from django.conf import settings
+from django.db.models import Q
+
+from signals.apps.services.domain.permissions.base import PermissionService as BasePermissionService
+
+
+class PermissionService(BasePermissionService):
```

Comment:
Should this not be `SignalPermissionService` it controls access to `Signal` instances after all?

Reply:
It is already in the permission/signal.py file, which for me also shows that this is a SignalPermissionService. I don't mind classes to have the same name. That said I also don't mind renaming this to SignalPermissionService.

Reply:
You are right, but we now have two things called `PermissionService` (as evidenced by the import line above). I would rather it be called `SignalPermissionService` so as to prevent confusion. That and if you have some other object that gets permissions it should not also end up being called `PermissionService`.

Reply:
It's just on how you see it. I do not have a strong opinion about 1 or the other. Changed it to SignalPermissionService.